### ISSUE TITLE
Add a gulp usage example to the readme with PostCSS

### ISSUE THREAD

**Issue by johnotander** [johnotander/immutable-css#27]

**Comment by marchdoe** [comment]
I would also be interested in a webpack example. I am not doing something right, when trying to integrate it into my webpack config.

**Comment by TrySound** [comment]
@marchdoe Use [postcss-loader](https://github.com/postcss/postcss-loader)

**Comment by TrySound** [comment]
@johnotander Use [gulp-postcss](https://github.com/postcss/gulp-postcss)

**Comment by marchdoe** [comment]
@TrySound - yeah, I guess I am just not sure how to integrate it into postcss-loader.

**Comment by TrySound** [comment]
@marchdoe Like any other plugin

From doc
```js
var mutations = postcss([import(), immutableCss(), reporter()])
                  .process(css, { from: 'styles.css' })
```

**Comment by TrySound** [comment]
@marchdoe 
In webpack case
```js
var immutable = require('immutable-css');

module.exports = {
    module: {
        loaders: [
            {
                test:   /\.css$/,
                loader: "style-loader!css-loader!postcss-loader"
            }
        ]
    },
    postcss: function () {
        return [immutable, precss];
    }
}
```

**Comment by marchdoe** [comment]
@TrySound thanks, that last part helped me find my issue. Appreciate the quick response.

**Comment by twoism-dev** [comment]
Can be closed as resolved in PR #36.
